### ISSUE TITLE
iso 8601 for date and time formatting?

### ISSUE THREAD

**Issue by ThomasWaldmann** [vector-im/riot-web#3101]
May I suggest to use the iso way for formatting dates and times?

https://xkcd.com/1179/

Currently, e.g. the "last used" value for the devices (settings dialogue) looks like this:
```
myip @ 2/1/2017, 12:10:34 AM
```
... which is confusing for non-american non-12h clock users. iso-like formatted it would look like this:
```
myip @ 2017-02-01 00:10:34
```
(which is also the way it sorts naturally).

**Comment by turt2live** [comment]
For complete clarity, words can also help. I've personally never been a fan of entirely-numeric dates.

`ip @ Feb 1, 2017 12:10:34 AM`

**Comment by ThomasWaldmann** [comment]
@turt2live Words need translations, numbers don't. Also, you still have strange 12h clock.

**Comment by turt2live** [comment]
Translations haven't been implemented yet as is (although that's not an excuse to have words). https://github.com/vector-im/riot-web/issues/680

I also personally prefer AM/PM. 

This sounds like another setting to me - as preferences differ between regions and people.

**Comment by ThomasWaldmann** [comment]
Yes, if one has date time setting preferences, one can of course offer dozens of ways to format them (see xkcd picture). If one has not, the best way to go is iso standard.

**Comment by croulibri** [comment]
I would support the idea of implementing international standards (ISO) in order to avoid using a system based on preference of a limited number of people.
When translation comes, then maybe different format of date could be proposed.
So the proposed :  `myip @ 2017-02-01 00:10:34`
seems the more suitable to fit with the diversity of culture, language and practices of Matrix users.

**Comment by ara4n** [comment]
see also #1728. As much as I like iso 8601, its not the must human-legible format - i'd rather we gave people the option to format the timestamps as they desire.
